content/publication_resources/bibtex/lahti.bib

```diff
@@ -2,7 +2,7 @@
 @Article{Khalighi2024acm,
   author =	 {Moein Khalighi and Giulio Benedetti and Leo Lahti},
   title =	 {{FdeSolver, a Julia Package for Solving Fractional Differential Equations}},
```

Comment:
Here, do

{{FdeSolver}, a {Julia} Package for Solving Fractional Differential Equations}